ios/RCTCameraManager.m

```diff
@@ -123,7 +128,7 @@ - (id)init {
   if ((self = [super init])) {
 
     self.session = [AVCaptureSession new];
-    self.session.sessionPreset = AVCaptureSessionPresetHigh;
+    self.session.sessionPreset = AVCaptureSessionPresetHigh; 
```

Comment:
thanks for the comments @pjcabrera . I understand this things don't make much sense - this is still WIP and I was just making sure my code was one the one breaking the projec